go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/grafana/terraform-provider-grafana
 
 go 1.18
 
+replace github.com/grafana/grafana-api-golang-client => /Users/elainevuong/go/src/github.com/grafana/grafana-api-golang-client
```

Comment:
TODO: needs to be removed upon final commit

Reply:
You can do a replace with your actual branch from github and that way we'll see if tests here run correctly